CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.6.0
```

Comment:
I think that the PR title is misleading: it should say "refactoring to use streams" or something like this and the changelog does not necessarily need to mention this (because it is not a change for the user), but any implications that this has should be mentioned. I cannot remember if there are any. If not, it's fine.